docs/data/material/getting-started/example-projects/example-projects.md

```diff
@@ -8,17 +8,7 @@ You can find some example projects in the [GitHub repository](https://github.com
 
 <!-- #default-branch-switch -->
 
-- [Next.js](https://github.com/mui/material-ui/tree/master/examples/material-next) ([TypeScript version](https://github.com/mui/material-ui/tree/master/examples/material-next-ts))
-- [Create React App](https://github.com/mui/material-ui/tree/master/examples/material-cra) ([TypeScript version](https://github.com/mui/material-ui/tree/master/examples/material-cra-ts))
-- [Remix](https://github.com/mui/material-ui/tree/master/examples/material-remix-ts)
-- [Gatsby](https://github.com/mui/material-ui/tree/master/examples/material-gatsby)
-- [Preact](https://github.com/mui/material-ui/tree/master/examples/material-preact)
-- [CDN](https://github.com/mui/material-ui/tree/master/examples/material-via-cdn)
-- [Express.js (server-rendered)](https://github.com/mui/material-ui/tree/master/examples/material-express-ssr)
-- [Tailwind CSS](https://github.com/mui/material-ui/tree/master/examples/material-cra-tailwind-ts)
-- [Vite.js](https://github.com/mui/material-ui/tree/master/examples/material-vite) ([TypeScript version](https://github.com/mui/material-ui/tree/master/examples/material-vite-ts))
-- [Use styled-components as style engine](https://github.com/mui/material-ui/tree/master/examples/material-cra-styled-components) ([TypeScript version](https://github.com/mui/material-ui/tree/master/examples/material-cra-styled-components-ts))
-- [Next.js + @mui/styles (v4 migration helper)](https://github.com/mui/material-ui/tree/master/examples/material-next-ts-v4-v5-migration)
+{{"demo": "ExampleCollection.js", "hideToolbar": true, "bg": "inline"}}
 
 Create React App is an awesome project for learning React.
```

Comment:
@samuelsycamore Does it make sense to keep this piece right here?

Reply:
Yeah, definitely agree! Would you remove the paragraph below it as well ("The source code...")? If we were to use our own documentation as an example, given it might have some interesting insights about how to use Material UI + Next, as a reader, I'd probably like something _a bit_ more fleshed out... What do you think?

Reply:
@danilo-leal I think I would just cut all of the text below. In fact, as I review the "Advanced examples," I'm not really sure they belong on this page at all. It feels like different content altogether. Plus, the latter two "Paid" examples appear to still be on v4 😭 so we probably don't want to direct new users to those.

Reply:
Haha, yes, that makes sense to me, too! I'll keep it there, though, for the sake of keeping this PR's scope contained but excited to see you revamp this content!